framework/include/utils/RankTwoTensor.h

```diff
@@ -369,6 +369,12 @@ class RankTwoTensorTempl : public TensorValue<T>
   /// Print the rank two tensor
   void print(std::ostream & stm = Moose::out) const;
 
+  /// Print the Real part of the DualReal rank two tensor
+  void printReal(std::ostream & stm = Moose::out) const;
```

Comment:
How about we fold all this into the original print method by adding a second default parameter `unsigned int nDual = 0`?

Reply:
ok, never mind, don't change anything now